### ISSUE TITLE
Won't pass, even though correct.  Doesn't recognize the + operator as being used

### ISSUE THREAD

**Issue by WebCodeNinja** [FreeCodeCamp/FreeCodeCamp#5898]
Challenge [Waypoint: Concatenating Strings with Plus Operator](http://www.freecodecamp.com/challenges/waypoint-concatenating-strings-with-plus-operator#?solution=%2F%2F%20Example%0Avar%20ourStr%20%3D%20%22I%20come%20first.%20%22%20%2B%20%22I%20come%20second.%22%3B%0A%0A%2F%2F%20Only%20change%20code%20below%20this%20line%0A%0Avar%20myStr%20%3D%20%22This%20is%20the%20start.%22%20%2B%20%22%20This%20is%20the%20end.%22%3B%0A%0A%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
// Example
var ourStr = "I come first. " + "I come second.";

// Only change code below this line

var myStr = "This is the start." + " This is the end.";



```

**Comment by WebCodeNinja** [comment]
Fails the second test even though it is correctly using the + operate as far as I can see.

**Comment by sludge256** [comment]
It's testing for a space at end of `"This is the start."` instead of testing the end result for a space.
```js
var myStr = "This is the start." + " This is the end.";
```
Should be 
```js
var myStr = "This is the start. " + "This is the end.";
```
It's easily missed unless you pay close attention to the strings given to you in the instructions.

**Comment by WebCodeNinja** [comment]
If that passes great, but the end result string was already passing the test. It was the second test "use the + operator to concatenate the strings" that was failing.  Will close, but I still think it might need some work.

**Comment by jojo3000** [comment]
I works only with single quotes i found !!

**Comment by jorycn** [comment]
Yeah, it works only with single quotes!

**Comment by peterallenio** [comment]
I had the correct answer and it was failing the same test "use the + operator to concatenate the strings" (Yes, I had the spaces in the correct place). The issue was resolved when I extended the `var myStr;` instead of leaving `var myStr;` and adding `myStr = [code];` on a new line. 

I prefer to only write new code and avoid editing existing code when I can help it. If you want people to edit existing variables then it might be worth explicitly stating that (especially if it will cause the test to fail for an incorrect reason), alternatively improve the error testing so we know exactly what's wrong. It's quite frustrating to have the correct answer but be stuck for five minutes trying to figure multiple ways of getting it to pass until you strike gold.
